pkg/module_manager/module_manager_hooks.go

```diff
@@ -80,8 +84,13 @@ func (mm *ModuleManager) registerGlobalModule(globalValues utils.Values) error {
 		MetricStorage:      mm.dependencies.MetricStorage,
 	}
 
-	gm := modules.NewGlobalModule(mm.GlobalHooksDir, globalValues, mm.ValuesValidator, &dep)
+	gm, err := modules.NewGlobalModule(mm.GlobalHooksDir, globalValues, &dep, configBytes, valuesBytes)
+	if err != nil {
+		return fmt.Errorf("new global module: %w", err)
+	}
+
 	mm.global = gm
+	log.Infof(gm.GetSchemaStorage().GlobalSchemasDescription())
```

Comment:
Is it some debug info?

Reply:
Yes, but this line was [there](https://github.com/flant/addon-operator/pull/470/files/dae21fbd4c189252ef4a591f88b9cfa39c2b921f#diff-e660fb6fddb6a937f54c49904c858dfd52b579e5a4a9fa414e046976b75a6fb4L73) earlier, so I decided to leave it